docs/getting-started/index.mdx

```diff
@@ -222,6 +222,8 @@ npm run build
 
 # Serving Frontend with the Backend
 cd ./backend
+python -m venv
+./venv/Scripts/activate
```

Comment:
This is not universally applicable across platforms, and there are many different ways to VirtualEnv. I think a note above the instruction would be better.

Reply:
updated, is this ok?

Reply:
by the way

cp -RPp .env.example .env

this command fails on windows

Reply:
#### Build and Install 🛠️

Run the following commands to install:

```sh
git clone https://github.com/open-webui/open-webui.git
cd open-webui/

# Copying required .env file
cp -RPp .env.example .env

# Building Frontend Using Node
npm i
npm run build
```

### Serving Frontend with the Backend

Navigate to the backend directory, then follow the instructions for your operating system.

#### For Linux/macOS:

```sh
cd ./backend

# Optional: You can enable a virtual environment to avoid collisions with system-wide packages
python3 -m venv venv
source venv/bin/activate

# Install dependencies
pip install -r requirements.txt -U

# Start the application
bash start.sh
```

#### For Windows:

```cmd
cd .\backend

:: Optional: You can enable a virtual environment to avoid collisions with system-wide packages
python -m venv venv
venv\Scripts\activate

:: Install dependencies
pip install -r requirements.txt -U

:: Start the application
start.bat
```

You should have Open WebUI up and running at [http://localhost:8080/](http://localhost:8080/). Enjoy! 😄